lib/overrides.js

```diff
@@ -26,13 +24,11 @@ const KNWON_DORMANT_ADDONS = Object.freeze([
 ]);
 module.exports = class Overrides {
   static for(project, env = process.env) {
-    if (env.EMBER_ENV === 'production') {
-      return new Overrides([]);
-    } else {
-      return new Overrides(Overrides.addonsInfoFor(project), {
-        showAllEmberGlobalDeprecations: env.EMBER_GLOBAL_DEPRECATIONS === 'all',
-      });
-    }
+    return env.EMBER_ENV === 'production'
+      ? new Overrides([])
+      : new Overrides(Overrides.addonsInfoFor(project), {
+          showAllEmberGlobalDeprecations: env.EMBER_GLOBAL_DEPRECATIONS === 'all',
+        });
```

Comment:
Similarly, while I'm a fan of ternaries and expression-oriented programming, this isn't actually an *improvement* in a strict sense. It's just *different*. Better to leave it unchanged!